tests/test_fiqci.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021-2023 IQM client developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for changes to the IQM client in the context of the FiQCI project.
+"""
+import os
+
+from iqm.iqm_client import IQMClient, update_batch_circuit_metadata
+
+
+def test_iqm_client_initializes_with_project_id(base_url):
+    """Test that IQMClient initializes successfully when project ID is available as environment variable."""
+    os.environ['PROJECT_ID'] = 'ABC123'
```

Comment:
Thanks for checking!

I want it to be self-contained for now. The data shouldn't leak into other future tests. The proper way to do this would be to introduce a fixture, but since we only have two tests I don't see it as necessary.

Reply:
Also handle the fact that SLURM_JOB_ACCOUNT might be unset, if the code is run from a login node.

Reply:
The rejection of the job will be done on our side if `PROJECT_ID` is not set. Afaik `SLURM_JOB_ACCOUNT` is exposed as `PROJECT_ID` (@TheKharleeci can you confirm?)